sdformat_urdf/test/joint_tests.cpp

```diff
@@ -28,9 +27,21 @@ TEST(Joint, joint_ball)
   sdf::Errors errors;
   urdf::ModelInterfaceSharedPtr model = sdformat_urdf::parse(
     get_file(PATH_TO_SDF_JOINT_BALL), errors);
-  EXPECT_FALSE(errors.empty());
+  EXPECT_TRUE(errors.empty()) << errors;
   EXPECT_NO_ALGORITHM_ERRORS(errors);
-  ASSERT_FALSE(model);
+  ASSERT_TRUE(model);
+  ASSERT_EQ("joint_ball", model->getName());
+
+  urdf::JointConstSharedPtr joint = model->getJoint("joint_ball");
+  ASSERT_NE(nullptr, joint);
+
+  EXPECT_EQ("joint_ball", joint->name);
+  EXPECT_EQ(urdf::Joint::FLOATING, joint->type);
+  ASSERT_EQ(nullptr, joint->dynamics);
+  ASSERT_EQ(nullptr, joint->limits);
+  ASSERT_EQ(nullptr, joint->safety);
+  ASSERT_EQ(nullptr, joint->calibration);
+  ASSERT_EQ(nullptr, joint->mimic);
```

Comment:
I think lines 40!44 can be `EXPECT_EQ`, because we're not trying to access those pointers in the test anyway. This way, if in the future one of them fails, we will know whether the others fail too.

Reply:
I agree, though this could already be improved about the existing tests in this file so I wouldn't block on it.

Reply:
I can work on this in a separate PR

Reply:
I can work on this in a separate PR :)

Reply:
#16